docs/framework/react/api/router/useBlockerHook.md

```diff
@@ -7,19 +7,57 @@ The `useBlocker` method is a hook that [blocks navigation](../../guide/navigatio
 
 ## useBlocker options
 
-The `useBlocker` hook accepts a single _optional_ argument, an option object:
+The `useBlocker` hook accepts a single \_required argument, an option object:
 
-### `options.blockerFn` option
+### `options.shouldBlockFn` option
 
-- Optional
+- Required
 - Type: `BlockerFn`
-- The function that returns a `boolean` or `Promise<boolean>` indicating whether to allow navigation.
+- This function should return a `boolean` or a `Promise<boolean>` that tells the blocker if it should block the current navigation
+- The function has the argument of type `BlockerFnArgs` passed to it, which tells you information about the current and next route and the action performed
+- Think of this function as telling the router if it should block the navigation, so returning `true` mean that it should block the navgation and `false` that it should be allowed
+
+### `options.disabled` option
+
+- Optional - defaults to `false`
+- Type: `boolean`
+- Specifies if the blocker should be entirely disabled or not
 
-### `options.condition` option
+### `options.enableBeforeUnload` option
```

Comment:
same here. you can add a new property, mark the old one as deprecated but still accept and use it if is presented